README.md

```diff
@@ -1,4 +1,14 @@
-# Python boilerplate for GitHub tasks
+# Taxi service authentication 
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
+
+In this task, you will implement a visit counter, login/logout functionality, and make your site available only 
+authenticated users.
+
+1. Implement a visit counter on the home screen. It should show how many times a user visited the home page before.
+
+2. Create login/logout screens. Provide authentication to your project using built-in Django authentication.
+
+3. Display the username on the top of the sidebar and login/logout button depending on if the user is authenticated.
```

Comment:
If the driver clicks on `username` - the corresponding Driver detail page must open.